### PR TITLE
support journal entries

### PR DESCRIPTION
**Pull request by Sravanksk**

### CODE REVIEW

**Comment by Sravanksk** [comment]
Tests - 
integration_tests
internal_tests

File: test/integration/test_journal_entries.py [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+import pytest
+import json
+import os
+
+logger = logging.getLogger(__name__)
+
+def test_get(nc):
+    data = next(nc.journal_entries.get_all_generator())
+    logger.debug('data = %s', data)
+    assert data, 'get all generator didnt work'
```
Comment (snarayanank2) [inline test/integration/test_journal_entries.py:11]: can you assert more things about the result to ensure that it works as expected?

File: test/integration/test_journal_entries.py [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+
+    internal_id = data['internalId']
+    data = nc.journal_entries.get(internalId=internal_id)
+    logger.debug('data = %s', data)
+    assert data, f'No object with internalId {internal_id}'
+
+def test_post(nc):
+    filename = os.getenv('NS_ACCOUNT').lower() + '.json'
```
Comment (snarayanank2) [inline test/integration/test_journal_entries.py:19]: make NS_ACCOUNT a fixture.. the fixture function should read from os.getenv

File: test/internal/test_get.py [hunk 3]
```diff
@@ -12,6 +12,11 @@ def test_get_vendor_bill(ns):
     record = ns.get(recordType='vendorBill', externalId='1234')
     assert record, 'No vendor bill found'
 
+
+def test_get_journal_entry(ns):
+    record = ns.get(recordType='journalEntry', externalId='JE_01')
+    assert record, 'No journal entry found'
```
Comment (snarayanank2) [inline test/internal/test_get.py:18]: assert some more things

**Pull request merged**